file_formats/ie_formats/itm_v1.1.htm

```diff
@@ -10,19 +10,19 @@
     
     <div class="fileHeader">General Description</div>
     <div class="indent1">
-      This file format describes an &quot;item&quot;. Items include weapons, armor, books, scrolls, rings and more. Items can have attached abilties, occuring either when a target creature it hit, or when the item is equipped. ITM files have a similar structure to <a href="spl_v1.htm">SPL </a>files.<br />
+      This file format describes an &quot;item&quot;. Items include weapons, armor, books, scrolls, rings and more. Items can have attached abilties, occuring either when a target creature it hit, or when the item is equipped. ITM files have a similar structure to <a href="spl_v1.htm">SPL</a> files.<br />
     </div>
     <br />
     
     <div class="center"><img border="0" src="../../images/itm.gif"></div>
     <table align="center">
       <tr>
         <td><img src="../../images/red.jpg" width="10" height="10"></td>
-        <td><strong>Header:</strong> <font size="-1">Size=114 Bytes</font></td>
+        <td><strong>Header:</strong><font size="-1">Size=114 Bytes</font></td>
         <td><img src="../../images/blue.jpg" width="10" height="10"></td>
-        <td><strong>Ext. Header:</strong> <font size="-1">Size= 56 Bytes</font></td>
+        <td><strong>Ext. Header:</strong><font size="-1">Size= 56 Bytes</font></td>
         <td><img src="../../images/green.jpg" width="10" height="10"></td>
-        <td><strong>Feature Block:</strong> <font size="-1">Size=48 Bytes</font></td>
```

Comment:
Why these removals? I would understand fixing the different spacing around the equals sign, but this will just make the label less readable — no space between the colon and the text.

Reply:
Same thing for other versions, of course. You can just remove the font tag too, since it does nothing.

Reply:
I didn't catch that those spaces were textual.  Will fix.  I may look at removing/replacing depreciated tags like font in a later commit.

BTW, what is the SOP on my end after implementing your suggestions on the PR?  Make a new commit and new PR?

Reply:
You can commit to the same branch and push — I can always use the squash merging option. If you want to fix the existing commit, do so (commit --amend) and then force push (push -f) the branch.